README.md

```diff
@@ -12,4 +14,38 @@ The goal of this project would be to create a tool that intuitively lets users u
 
 ## Starting the Project
 
-Clone this repo, then from the terminal run `npm install` to install the dependencies then `npm start`. This will start a live-reloading server.
+Clone this repo, then from the terminal run
+
+```js
```

Comment:
maybe worth splitting this in 2 different code blocks. Also this multi-line code block says its `js` but it's not. Maybe write:

`npm install` or `yarn install`.